repo-projects/diagnostics.proj

```diff
@@ -2,6 +2,7 @@
 
   <ItemGroup>
     <RepositoryReference Include="arcade" />
+    <RepositoryReference Include="runtime" />
```

Comment:
Just curious, why is this new repo dependency necessary? This has implications on build ordering.

Reply:
I see. This change still needs to be undone.